tests/by-util/test_install.rs

```diff
@@ -326,6 +326,7 @@ fn test_install_copy_file() {
 
 #[test]
 #[cfg(target_os = "linux")]
+#[ignore]
```

Comment:
why ignore?
(and no comment to explain why)

Reply:
https://github.com/uutils/coreutils/pull/1811/commits/f7f421387d3e69a50a4f9719802974f89af70144#diff-28f964575c1020d23413d29689d0eca826f901cbd166445db2ef984b660540f5R329